### ISSUE TITLE
Location_information (Priority = High)

### ISSUE THREAD

**Issue by sadiekelly** [globaldothealth/outbreak-schema#6]
*Minimum EpiCore variables Neighbourhood of residence, Address of residence, Town or city of residence, Administrative level 4/3/2/1/0 of residence

The day0 schema collects location data based upon where the case was reported from.

The T0 toolkit indicates location is based upon the subjects usual residence, which may not be available from G.h sources. In the case that the subjects' residence data is available, additional attributes may be required in the day0 schema. 

Ensure that the report location and the residence location do not become aligned when coding as the meaning differs.

**Comment by lauramerson** [comment]
Agree. The day0 variable should be clearly defined as reporting location to differentiate.
I expect G.h's DPIA will have significant limitations on subject residence data. Town/city and corresponding administrative levels could be added if through relevant to a particular outbreak.

**Comment by sadiekelly** [comment]
The day0 schema should capture both location of report and location of residence as separate variables. 
Amend existing variable Location_information to Location_information_report and create new variable Location_information_residence.

**Comment by sadiekelly** [comment]
@aimeehan1 @JacqSauer

**Comment by aimeehan1** [comment]
@julianasopko

**Comment by aimeehan1** [comment]
The way sources would display location data differed by country, by report, and changed over time.  There was no global standard. 

Some weeks data was displayed in a paragraph narrative, some weeks it was in a table, other weeks it was an (unlabeled) country map! A curator would have to compare a previous report(s) to a current report to calculate the delta in cumulative cases to get the number of incident cases for a specific location. 

If case data was displayed on a map - a curator would have to Google a location map with corresponding administrative boundaries to the one being used by the source and identify location names. We saw this in [France](https://www.santepubliquefrance.fr/les-actualites/2022/cas-de-variole-du-singe-point-de-situation-au-9-aout-2022
), Brazil, and other countries. See Slack conversation: https://ghdsi.slack.com/archives/C0115SR6V6E/p1658939357668659

You'll also notice in [France's](https://www.santepubliquefrance.fr/les-actualites/2022/cas-de-variole-du-singe-point-de-situation-au-9-aout-2022) report that they displayed cases on maps by Region of Residence, by Reporting Region and were inconsistent in this format from week to week. It was a challenge to disentangle cases and make sure we weren't underreporting or double counting.  Some weeks we'd have to skip an update and wait for the next report to be released to reconcile case or location differences.

**Comment by sadiekelly** [comment]
thank you @aimeehan1 ! I definitely appreciate the difficulties in curation here!
Region of residence was added to our schema as a possibility from the core set of epi variables and I think important for us to capture this information so we can see distance travelled to healthcare. It could also be important where a person is a resident of a country different to the one the case was reported from, to make it clear when reconciling case numbers from a country that a person from another country is included. Reporting region (to whatever granularity possible from the specific report), would always be provided. Residence location (country/state/district/city) could be captured separately if available  - what do you think?

**Comment by sadiekelly** [comment]
Residence location can be added to the schema but may not be frequently available, particularly when disentangling aggregated case reports. 
Residence location can be geolocated similar to report location.

**Comment by aimeehan1** [comment]
@sadiekelly Sure -- let's add a column to collect Residence location with the understanding for limitations and assumptions.

**Comment by sadiekelly** [comment]
Recommendation: Residence location can be added to the schema but may not be frequently available, particularly when disentangling aggregated case reports.
Residence location can be geolocated similar to report location.
